### ISSUE TITLE
Proxychain for removing restrictions from College LAN

### ISSUE THREAD

**Issue by chetan1999** [rofl0r/proxychains-ng#204]
Hey everyone, 

I am trying to remove the restrictions on my college LAN. My college LAN works after putting a https proxy in my computers proxy settings. ( I am using Macbook Air - Mac Os High Sierra )
So I thought of making a http server using squid and then using proxychains for removing the restrictions. So I have now two proxies one is of the college and other one is mine! But when I am trying to test it on my college's LAN using the command curl it is not working. While on my home it works perfectly fine. Any leads how can I make it work ? Is there is a possibility that something is wrong with the configuration my squid server ?

**Comment by rofl0r** [comment]
yes, that is possible. you need to configure squid to use the school proxy as upstream proxy. you can test if it works by using e.g. firefox and put your local proxy in the network settings.
